### DISCUSSION TITLE
官方提供的数码管识别模型问题

### DISCUSSION THREAD

**Discussion by Cupcc** [PaddlePaddle/PaddleOCR#14126] [Q&A]
# 问题描述
模型为什么没有识别出来？

识别结果： test_images/img.png	{"Student": {"label": "", "score": 0.0}, "Teacher": {"label": "", "score": 0.0}}

识别的图片： 
![image](https://github.com/user-attachments/assets/976f6def-7f35-4f24-8999-b400871ac588)

# 执行的操作
```
git clone -b release/2.6.1 --single-branch https://github.com/PaddlePaddle/PaddleOCR.git
pip install -r requirements.txt
```
识别推理命令
```bash
python3 tools/infer_rec.py -c pretrain_models/v3_digital_mobile/config.yml -o Global.pretrained_model=pretrain_models/v3_digital_mobile/best_accuracy  Global.infer_img="ppocr_img/img.png"
```

模型来自于百度网盘下的数码管模型 `v3_digital_mobile.tar`
字典文件
```txt
-
.
0
1
2
3
4
5
6
7
8
9
B
E
F
H
L
N
T
W
d
k
m
n
o
z
```
配置文件：解压包下的 `config.yml`

# 系统环境
```bash
OS: ubuntu 22.04 LTS
python: 3.10
# pip list
paddleocr              2.9.1
paddlepaddle           2.6.2
```

**Comment by UserWangZz** [comment]
这个是一个范例教程里的模型吧，尝试自己微调一下推理

**Reply by Cupcc** [reply]
不是微调的问题，是代码问题，2.6.0可以运行2.6.1之后的版本都不行。谁能解释一下为什么

**Reply by GreatV** [reply]
这个就是一个示例用于展示在垂类微调的过程，实际使用需要根据自己的场景微调。可以试试迁移到实际微调场景。

**Reply by GreatV** [reply]
这个示例是基于旧版paddleocr，如果想使用就切换到旧版吧。

**Reply by Cupcc** [reply]
> 这个示例是基于旧版paddleocr，如果想使用就切换到旧版吧。

希望能在案例中进行说明，不然搞了半天才发现是代码版本问题。

**Reply by SWHL** [reply]
@Cupcc 已经添加了相关说明，详情参见：https://github.com/PaddlePaddle/PaddleOCR/pull/14338
